ERCS/erc-XXXX.md

```diff
@@ -0,0 +1,254 @@
+---
+eip: XXXX
+title: Tamperproof Web Immutable Transaction (TWIT)
+description: Provides a mechanism for DAP to use the API defined in EIP-1193 in a tamperproof way
+author: Erik Marks, Guillaume Grosbois
+discussions-to: TODO
```

Comment:
Please create a topic in Eth Magicians.  https://ethereum-magicians.org/
It can just contain a link to this PR.

Reply:
will do.